### PR TITLE
OP-544 remove JRadioButton shortcut keys

### PR DESCRIPTION
**Pull request by dbmalkovsky**
See OP-544

Also added ".btn" suffix to the various JRadioButton key values.

### CODE REVIEW

File: bundle/language_en.properties [hunk 1]
```diff
@@ -321,6 +323,10 @@ angal.billbrowser.today.btn
 angal.billbrowser.today.btn.key                                                  = T
 angal.groupsbrowser.groupmenu.btn                                                = GroupMenu
 angal.groupsbrowser.groupmenu.btn.key                                            = M
+angal.lab.n.btn                                                                  = N
+angal.lab.p.btn                                                                  = P
```
Comment (mwithi) [inline bundle/language_en.properties:327]: ok, these are "negative" and "positive" abbeviations

File: bundle/language_en.properties [hunk 2]
```diff
@@ -321,6 +323,10 @@ angal.billbrowser.today.btn
...
+angal.labnew.ip.btn                                                              = IP
+angal.labnew.opd.btn                                                             = OPT
```
Comment (mwithi) [inline bundle/language_en.properties:329]: Fix value to "OPD"

**Review by mwithi — CHANGES_REQUESTED** [review]
little changes

File: bundle/language_en.properties (hunk 2 above)
Comment (dbmalkovsky) [reply bundle/language_en.properties:329]: I'll fix it but the original value in the bundle was "OPT"

File: bundle/language_en.properties (hunk 1 above)
Comment (dbmalkovsky) [reply bundle/language_en.properties:327]: Done

File: bundle/language_en.properties (hunk 2 above)
Comment (mwithi) [reply bundle/language_en.properties]: we just had chance to fix it 😉 thanks!

**Pull request merged**